### PR TITLE
Warn against mixing different versions of Framer Motion

### PR DESCRIPTION
**Pull request by mattgperry**
Fixes https://github.com/framer/motion/issues/477

### CODE REVIEW

File: packages/framer-motion/rollup.config.js [hunk 1]
```diff
@@ -7,6 +7,21 @@ const config = {
     input: "lib/index.js",
 }
 
+const replaceSettings = (env) => {
+    const replaceConfig = env
+        ? {
+              "process.env.NODE_ENV": JSON.stringify(env),
+              preventAssignment: false,
```
Comment (shuangq) [inline packages/framer-motion/rollup.config.js:14]: Shouldn't this be `true`? 🤔

**Review by shuangq — APPROVED** [review]

**Comment by fturmel** [comment]
@mattgperry this new feature doesn't seem to work correctly for me. I just upgraded a Next project and am getting this console warning: `Attempting to mix Framer Motion versions 6.3.9 with 6.3.9 may not work as expected.`

**Comment by mattgperry** [comment]
@fturmel Thanks for letting me know - I'm just published a quick fix in 6.3.10, will be a few min

**Comment by fturmel** [comment]
@mattgperry No problem. Working on my first project with Framer and enjoying it so far, keep up the great work!

**Pull request merged**